tests/UI/data/types/contactUs.ts

```diff
@@ -0,0 +1,8 @@
+type ContactUsData = {
+  subject: string,
+  message: string,
+  emailAddress: string,
+  reference: string,
+};
+
+export default ContactUsData;
```

Comment:
change this file to a definition file like `contactUs.d.ts`

Reply:
Fixed in a51e0e4